rust/src/api.rs

```diff
@@ -9,12 +9,22 @@ pub struct Balance {
     pub confirmed: u64,
 }
 
+pub struct Address {
```

Comment:
❓ Why do we need to wrap a single attribute into a struct. Can't we just return a String in the api?

Reply:
Usually I would argue: It's not bad to wrap it in terms of extendability of the API without breaking it. If we e.g. want to add to expose a `network` field as part of the `get_address` API we can just add it without breaking changes when having an `Address` type. 

For the moment it really does not matter and returning a String would be fine - but it did not take much effort wrapping it with a type :D